scripts/exregional_bias_correction_o3.sh

```diff
@@ -381,7 +374,7 @@ EOF1
 
     cp ${DATA}/${NET}.${cycle}.max_*hr_o3_bc.*.grib2 ${COMOUT}
    
-    if [ "$SENDDBN" = "TRUE" ]; then
+    if [ "$SENDDBN" = "YES" ]; then
```

Comment:
In `ush/job_preamble.sh`, the NCO variables are changed from `YES/NO` to `TRUE/FALSE` (lines 79-84). Therefore, even though their inputs are YES/NO in `head.h` or `env-p1.h`, they should be TRUE/FALSE in the ex-scripts. Please do not change this line.